src/Components/PlaylistModal/PlaylistModal.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react'
+import { useVideo } from '../../Context/Video-Context';
+import { searchPlaylistsForID } from '../../ReusableFunctions/funcs';
+import Toast from '../Toast/Toast';
+import './PlaylistModal.css'
+
+function PlaylistModal({course, showModal, setShowModal }) {
+
+    const [modalInput, setModalInput] = useState("");
+    const [toast, setToast] = useState(false);
+    const { state, dispatch } = useVideo();
+    
+    function addPlaylist(e) {
+        e.preventDefault();
+        if (modalInput.trim().length === 0)
+            return
+        dispatch({ type: "ADD_NEW_PLAYLIST", payload: modalInput })
+        setModalInput("");
+    }
+
+    function checkBoxHandler(item) {
+        if (searchPlaylistsForID(item.id, course.id) === true) {
```

Comment:
Since the function `searchPlaylistsForID` returns a boolean value, you can avoid writing `=== true`